buffer/src/main/java/io/netty/buffer/PooledUnsafeHeapByteBuf.java

```diff
@@ -129,6 +129,34 @@ protected void _setLongLE(int index, long value) {
     }
 
     @Override
+    public ByteBuf setZero(int index, int length) {
+        if (PlatformDependent.javaVersion() >= 7) {
```

Comment:
@normanmaurer would we be able to implement a "slow" version of this with `Unsafe` prior to 7?